ros2/nav2/pose_republisher/README.md

```diff
@@ -0,0 +1,10 @@
+# pose_republisher
+
+This node serves as a republisher for the `goal_pose` needed when using Foxglove in simulation.
+
+`foxglove_bridge` does not publish sim-timed poses.
```

Comment:
Why do you need this node? Because when publishing a pose from Foxglove, the pose's timestamp does not match the current time? Have you tried launching foxglove_bridge with `use_sim_time` set to true?

https://github.com/foxglove/ros-foxglove-bridge/blob/26d5fac4d11b34edc583ff63236910c94f4416e7/ros2_foxglove_bridge/launch/foxglove_bridge_launch.xml#L15

Reply:
Hello @achim-k,

I've verified that Foxglove does subscribe to the `clock` topic when launched with `use_sim_time:=true`. However the PoseStamped from the 3D panel is still being published with the epoch time

Reply:
Now it works! thank you for the support! I'll mark this issue as resolved.